### DISCUSSION TITLE
modifying questionnaire in a specific channel

### DISCUSSION THREAD

**Discussion by lbattistella1** [globaleaks/globaleaks-whistleblowing-software#4042] [General]
hi everyone. I'd need your help to fix (if possible) a problem that I am experiencing. I have two sites (the main and other two auxiliaries for two sister companies). I should modify part of the questionnaire of the sister companies without modifying the main site. Is it possible?
Thank you

**Comment by evilaliv3** [comment]
Sure, just clone the questionnaire using the clone function. This will create an independent copy or the questionnaire that you can use edit as you like
